packages/server/src/templates/templates.controller.ts

```diff
@@ -7,6 +7,7 @@ export class TemplatesController {
 
   @Get('/')
   getTemplate(): any {
-    return this.templateService.getTemplate();
+    const template_name = 'passwordReset';
```

Comment:
NIT: `templateName` instead of `template_name`.
Also may want to consider having the template type as a parameter/query provided to his endpoint so that it is easier to add on additional templates later on. Could then return some 4xx error if the provided template name doesn't exist